app/views/pages/_privacy_text.html.erb

```diff
@@ -0,0 +1,156 @@
+<p>Effective May 25, 2018</p>
```

Comment:
Pulled this out into a partial like coc and terms so it becomes easier to pass just the body to the new page adn remove the header fluff.